CMakeLists.txt

```diff
@@ -3,10 +3,22 @@ cmake_minimum_required(VERSION 3.10)
 set(LJMD_VERSION 1.0 CACHE STRING "Floating point number for the current version")
 project(ljmd VERSION ${LJMD_VERSION} LANGUAGES C)
 
-add_executable(ljmd.x main.c src/utilities.c src/input_output.c src/engine.c )
+add_executable(ljmd.x
```

Comment:
I think that short lines like this one are readable as-is and don't need breaking